src/helpers/utils.ts

```diff
@@ -175,9 +175,10 @@ export function filterProposals(space, proposal, tab) {
 
 export function infiniteScroll() {
   window.onscroll = () => {
+    console.log(window.screen.height);
```

Comment:
Can you remove this?
```suggestion
    console.log(window.screen.height);
```